include/gepetto/gui/shortcut-factory.hh

```diff
@@ -0,0 +1,60 @@
+#ifndef GEPETTO_GUI_SHORTCUT_FACTORY_HH_
+# define GEPETTO_GUI_SHORTCUT_FACTORY_HH_
+
+#include <QAction>
+#include <QKeyEvent>
+#include <QMouseEvent>
+#include <QPushButton>
+
+#include <map>
+#include <utility>
+
+namespace gepetto {
+  namespace gui {
+    class ShortcutFactory : public QObject {
+      Q_OBJECT
+    public:
+      ShortcutFactory();
+      ~ShortcutFactory();
+
+    public slots:
+      /// Add a binding to a given widget.
+      /// If widgetName doesn't exists, then it will be add.
+      /// If actionName exists, do nothing
+      /// \param widgetName name of the corresponding widget
+      /// \param actionName name of the action
+      /// \param action instance of the action
+      void addBinding(QString widgetName, QString actionName, QAction* action);
```

Comment:
At the beginning it was supposed to be a slot but then I changed my mind and forgot to change it.
I changed it.